docs/guides/src/main/server/configuration-metrics.adoc

```diff
@@ -0,0 +1,79 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Enabling Keycloak Metrics"
+summary="Learn how to enable and expose metrics from the server"
+includedOptions="metrics-enabled">
+
+Keycloak has built in support for metrics. This guide describes how to enable and configure server metrics.
+
+== Enabling Metrics
+
+It is possible to enable metrics using the build time option `metrics-enabled`:
+
+<@kc.start parameters="--metrics-enabled=true"/>
+
+== Querying Metrics
+
+Keycloak exposes metrics at the following endpoint:
+
+* `/metrics`
+
+The response from the endpoint uses a `text/plain` content type and it is based on the Prometheus text format. The snippet bellow
+is an example of a response:
+
+[source, json]
```

Comment:
Nit-pick: it is actually plain text, and not JSON. As the current guides don't do syntax highlighting, this doesn't matter in the output. So I'll still approve it.

```suggestion
[source]
```

Reply:
Done